infrastructure/hub-and-spoke-peerings/variables.tf

```diff
@@ -0,0 +1,121 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "hub_project_id" {
+  description = "Hub Project id. Same project can be used for hub and spokes."
+  type        = string
+}
+
+variable "spoke_1_project_id" {
+  description = "Spoke 1 Project id. Same project can be used for hub and spokes."
+  type        = string
+}
+
+variable "spoke_2_project_id" {
+  description = "Spoke 2 Project id. Same project can be used for hub and spokes."
+  type        = string
+}
+
+variable "hub_subnets" {
+  description = "Hub VPC subnets configuration."
+  type = map(object({
+    ip_cidr_range      = string
+    region             = string
+    secondary_ip_range = map(string)
+  }))
+  default = {
+    hub-subnet-a = {
+      ip_cidr_range      = "10.10.10.0/24"
+      region             = "europe-west1"
+      secondary_ip_range = {}
+    }
+    hub-subnet-b = {
+      ip_cidr_range      = "10.10.20.0/24"
+      region             = "europe-west2"
+      secondary_ip_range = {}
+    }
+  }
+}
+
+variable "spoke_1_subnets" {
```

Comment:
Same as for hub subnets. I would use a single variable for all IP ranges in the example, as it makes it much easier to work and keep track of the IP space.